geocoder/src/main/scala/grasshopper/geocoder/search/census/CensusGeocode.scala

```diff
@@ -38,15 +37,16 @@ trait CensusGeocode {
           val city = addressInput.city
           val state = f.get("STATE").getOrElse("")
           val zipCodeR = f.get("ZIPR").getOrElse("")
-          f.addOrUpdate("address", s"${addressNumber} ${streetName} ${city} ${state} ${zipCodeR}")
+          f.addOrUpdate("address", s"$addressNumber $streetName $city $state $zipCodeR")
```

Comment:
I guess this is really a style thing.  Less clutter.  The IntelliJ IDE was complaining about how the braces were unnecessary in this simple case, so I removed them.